target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -36,10 +37,30 @@
 
 	soc {
 		pinmux@800000 {
-			i2c4_pins: i2c4_pinmux {
-				pins = "gpio12", "gpio13";
-				function = "gsbi4";
-				bias-disable;
```

Comment:
@dissent1 is right. These gpios are defined in ipq806x_ap148_gpiomux_c2600 and therefore should be kept.